prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -109,63 +117,86 @@ func handlePullRequest(pc plugins.Agent, pre github.PullRequestEvent) error {
 	return handle(
 		pc.GitHubClient,
 		oc, pc.Logger,
-		pc.PluginConfig.Blunderbuss.ReviewerCount,
-		pc.PluginConfig.Blunderbuss.FileWeightCount,
-		pc.PluginConfig.Blunderbuss.MaxReviewerCount,
-		pc.PluginConfig.Blunderbuss.ExcludeApprovers,
-		&pre,
+		pc.PluginConfig.Blunderbuss,
+		&pre.Repo,
+		&pre.PullRequest,
+	)
+}
+
+func handleGenericCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent) error {
```

Comment:
Where do we check that the command is in the generic comment event?

Reply:
I actually forgot to add this part. 🙈